apps/web/src/config/index.ts

```diff
@@ -14,6 +14,11 @@ export const API_ROOT =
     ? window._env_.REACT_APP_API_URL || process.env.REACT_APP_API_URL || 'http://localhost:1336'
     : window._env_.REACT_APP_API_URL || process.env.REACT_APP_API_URL || 'http://localhost:3000';
 
+export const CMS_ROOT_URL =
+  window._env_.REACT_APP_CMS_ROOT_URL || isCypress
+    ? window._env_.REACT_APP_CMS_ROOT_URL || process.env.REACT_APP_CMS_ROOT_URL || 'http://localhost:1336'
+    : 'https://api.novu.co';
```

Comment:
This way all the clients including the self-hosted will point to the same server.